conan/tools/cmake/toolchain/blocks.py

```diff
@@ -854,13 +854,21 @@ def template(self):
 
         return textwrap.dedent("""
            set(CMAKE_INSTALL_PREFIX "{{package_folder}}")
+           {% if default_bin %}
```

Comment:
Could any of these directories be `""`? I mean, to specify the current directory we should write `"."`, right?

Reply:
Yes, if defined, at least a "." is necessary, the empty directory semantics is not very clear.